TOOLS/odoo-tools.sh

```diff
@@ -493,9 +495,19 @@ if [ "$SCRIPT_MODE" = "newdb" ]; then
         echo -e "ERROR: ${DBPATH} could not be created!"; exit 2
     fi
 
-    # ----- Create BACKUP directory
+    # ----- Create BACKUP directory and mount remote Backup location and Write fstab
     if  mkdir ${DBBACKUPPATH} 2>&1>/dev/null; then
         echo -e "Database BACKUP Directory ${DBBACKUPPATH} created."
+        read -p "Do you you want to create a remote BACKUP location at external Storage ${QNAPBACKUPSERVER} ? otherwise local dir is used. To continue: <y/N>" prompt
+        if [[ ${prompt} =~ [yY](es)* ]]; then
+            echo "creating remote BACKUP location and mount into ${DBBACKUPPATH}"
```

Comment:
General convention for echo: Please start with an uppercase letter.
```echo "Creating remot...```
No need to correct all of this here but please consider it for the future.